README.md

```diff
@@ -2,6 +2,9 @@
 
 This report allows access to all backups on a site - including those stored in the users private files area and can download/restore or delete the backup files.
 
+Andreas Schenkel: Added a feature to be able to activate the report be config.php
```

Comment:
thanks Andreas - we don't mantain a manual changelog in most of our plugins (the git history is the changelog) - can you please remove this line from the readme?